rocketpy/tools.py

```diff
@@ -381,6 +382,107 @@ def check_requirement_version(module_name, version):
     return True
 
 
+def is_float(element):
+    """
+    Returns a boolean indicating us if an element is convertible to a float or not.
+    True : the element is convertible to a float
+    False : the element is not convertible to a float
+
+    Parameters
+    ----------
+    element : any
+        This is the element to test.
+
+    Returns
+    -------
+    result : boolean
+        The element is convertible or not.
+    """
+    if element is None:
+        return False
+    try:
+        float(element)
+        return True
+    except ValueError:
+        return False
```

Comment:
I think you could avoid the first if statement here.

Please test it locally before accepting.

```suggestion
    try:
        float(element)
        return True
    except (ValueError, TypeError):
        return False
```

Reply:
Just tested it. You're right, it is not necessary to keep it.